languages/javascript/exercises/concept/array-transformations/.docs/introduction.md

```diff
@@ -0,0 +1,21 @@
+In Javascript, the Array class has many powerful built-in functions for transforming arrays. These functions make it much easier to do things than it otherwise would be using a simple for loop or more direct manipulation.
```

Comment:
```suggestion
In Javascript, the `Array` class has many powerful built-in functions for transforming arrays. These functions make it much easier to do things than it otherwise would be using a simple for loop or more direct manipulation.
```